src/definitions/behaviors/form.js

```diff
@@ -338,18 +338,18 @@ $.fn.form = function(parameters) {
           blank: function($field) {
             return String($field.val()).trim() === '';
           },
-          valid: function(field) {
+          valid: function(field, showErrors) {
```

Comment:
Should `showErros` be added to docs ?
https://fomantic-ui.com/behaviors/form.html#/settings

Reply:
yes, will do this as a docs PR. Tagging as "awating docs" Thanks ;)